com.microsoft.java.test.plugin/src/main/java/com/java/junit/plugin/internal/JUnitPlugin.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *     Microsoft Corporation - initial API and implementation
  *******************************************************************************/
-package com.java.junit.plugin.internal;
+package com.microsoft.java.test.plugin.internal;
```

Comment:
how do you make the change? the folder structure isn't synced with the package..